Treat Solaris as Linux, since we use GNU CC and GNU LD.

--- configure.ac
+++ configure.ac
@@ -931,7 +931,7 @@
 
 AC_ARG_ENABLE(dynamic, [  --enable-dynamic        enable dynamically loaded drivers (default=no)],
 [	case `uname` in
-		Linux*|GNU*)
+		Linux*|GNU*|SunOS)
 		INSTALL_SHARED="install-shared"
 		DYNAMIC_CFLAGS="-fPIC"
 		if test "x$X_DEVS" != x; then
